### PR TITLE
Convert webui/monitor views to HAML. Get rid of unused partials

### PR DESCRIPTION
**Pull request by mdeniz**
This was done automatically with https://rubygems.org/gems/html2haml (see https://github.com/haml/html2haml for how to). Some issues were fixed by hand.

### CODE REVIEW

File: src/api/app/views/webui/monitor/_building_table.html.haml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        %tr
+          %th Project
+          %th Package
+          %th Repository - Build Arch
+          %th Job time
+          %th Build Host
+          %th Host Arch
+      %tbody
+        - @workerstatus.elements("building").sort {|a,b| b["starttime"] <=> a["starttime"] }.each do |building|
+          - project = building["project"]
```
Comment (Ana06) [inline src/api/app/views/webui/monitor/_building_table.html.haml]: Are this variables really needed? :thinking:

File: src/api/app/views/webui/monitor/_events.html.haml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+.grid_8.omega
+  #jobs{ style: "height:250px; margin-right: 20px" }
+  .centered
+    Packages scheduled for building
+    %span#legend_jobs.flot_legend
+  .plotspinner= image_tag('ajax-loader.gif')
+.clear
+
+= content_for :head_javascript do
+  :plain
```
Comment (Ana06) [inline src/api/app/views/webui/monitor/_events.html.haml]: What is this `:plain`? :confused:

File: src/api/app/views/webui/monitor/_stats.html.haml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+- if @workerstatus.elements("waiting").size == 0
```
Comment (Ana06) [inline src/api/app/views/webui/monitor/_stats.html.haml]:
``` Ruby
if @workerstatus.elements("waiting").empty?

File: src/api/app/views/webui/monitor/_workers_table.html.haml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  %span#workers_updating.hidden (updating...)
+%div
+  %p
+    This shows the single workers and their jobs. The
+    %em progress
+    shown (and color) is not for the time it will take (we don't know that before),
+    but just relative against each other. The exact percentage shown has no real meaning, just one thing is certain: the bar reaches its maximum at 4h.
+  %p
+    The monitor is meant to entertain and not to be exact, if you need to know more details, check the #{link_to 'detailed page', action: :old}.
+- if @workers_sorted.size == 0
```
Comment (Ana06) [inline src/api/app/views/webui/monitor/_workers_table.html.haml]:
``` Ruby
if @workers_sorted.empty?

File: src/api/app/views/webui/monitor/_building_table.html.haml (hunk 1 above)
Comment (mdeniz) [inline src/api/app/views/webui/monitor/_building_table.html.haml]: For readability... yes

File: src/api/app/views/webui/monitor/_events.html.haml (hunk 2 above)
Comment (mdeniz) [inline src/api/app/views/webui/monitor/_events.html.haml]: You can read the manual...  😸 
But, is for having plain text (keeping the indentation) inside the HAML, so we are using the `conten_for` helper for including javascript into a tag lately, so the best is using a :plain to include it there, if I use :javascript then I will end up having nested `<script>` tags

File: src/api/app/views/webui/monitor/_stats.html.haml (hunk 3 above)
Comment (mdeniz) [inline src/api/app/views/webui/monitor/_stats.html.haml]: I don't care that much... I'm trying not to refactor that much...

File: src/api/app/views/webui/monitor/_workers_table.html.haml (hunk 4 above)
Comment (mdeniz) [inline src/api/app/views/webui/monitor/_workers_table.html.haml]: Same here... if you have a strong opinion...

**Comment by mdeniz** [comment]
Take a look at the title of this PR, "...Get rid of unused partials..." so, I removed the files because the are not longer used.... 😄

File: src/api/app/views/webui/monitor/_workers_table.html.haml (hunk 4 above)
Comment (Ana06) [inline src/api/app/views/webui/monitor/_workers_table.html.haml]: It should be changed, but It can be done afterwards as it is really unrelated to the PR

**Pull request merged**